extension/src/experiments/model/collect.test.ts

```diff
@@ -7,15 +7,15 @@ import {
 } from '../../cli/dvc/contract'
 
 describe('collectExperiments', () => {
-  it('should return an empty array if no branches are present', async () => {
```

Comment:
Looks like these unneeded `async` and `await`s are leftover from #3035.

Reply:
can we turn this on: https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/await-thenable.md?